tiles/src/test/java/com/protomaps/basemap/layers/PoisTest.java

```diff
@@ -34,4 +34,43 @@ void busStop() {
         0
       )));
   }
+
+  @Test
+  void allotments() {
+    assertFeatures(15,
+      List.of(Map.of("pmap:kind", "allotments"), Map.of("pmap:kind", "allotments", "pmap:min_zoom", 13, "name", "Kleingartenverein Kartoffel")),
+      process(SimpleFeature.create(
+        newPolygon(0, 0, 0, 1, 1, 1, 1, 0, 0, 0),
+        new HashMap<>(Map.of("landuse", "allotments", "name", "Kleingartenverein Kartoffel")),
+        "osm",
+        null,
+        0
+      )));
+  }
+
+  @Test
+  void villageGreen() {
+    assertFeatures(15,
+      List.of(Map.of("pmap:kind", "village_green"), Map.of("pmap:kind", "village_green", "pmap:min_zoom", 8, "name", "Stadtpark Eiche")),
+      process(SimpleFeature.create(
+        newPolygon(0, 0, 0, 1, 1, 1, 1, 0, 0, 0),
+        new HashMap<>(Map.of("landuse", "village_green", "name", "Stadtpark Eiche")),
+        "osm",
+        null,
+        0
+      )));
+  }
+
+  @Test
+  void playground() {
+    assertFeatures(15,
+      List.of(Map.of("pmap:kind", "playground"), Map.of("pmap:kind", "playground", "pmap:min_zoom", 13, "name", "Spielwiese")),
```

Comment:
I am a bit unsure about why I see two elements here - maybe just have a closer look whetehr the tests are what you would expect :-)

Reply:
@bdon can you comment on this question about test construction, please?

Reply:
The tests run against the entire Profile, not just a single layer class, so you're also getting the landuse element in the result set.

Reply:
Okay, understood, but that means I'm basically not unit-testing the part I'm interested in. I don't necessarily want to refactor it so it is independent - I think it's rather up to you how you want this part to evolve in the future. I'll add a comment about what is happening.